packages/Liquid-Core.package/LiquidPoll.class/instance/startWithId.andPassword.ifTaken..st

```diff
@@ -0,0 +1,13 @@
+initialize
+startWithId: anId andPassword: aPassword ifTaken: do
+	self id: anId;
+		startTime: DateAndTime now.
+	guard := LQPasswordGuard newWithPassword: aPassword.
```

Comment:
[question] `guard` is an instance variable of Poll without getter and setter. Is this like a runtime `private`? So we can set it here, but cannot change it via the messages we send over the network?

[mountain] Could you change the guard by calling this function again, with the same id and your own password?

Reply:
I updated it so that passwords can't be overriden: https://github.com/hpi-swa-teaching/Liquid/pull/121/commits/c9d35cf587af52a7e9fe460443e30927b3f3a664